id: PYSEC-2020-32
package:
  name: django
  ecosystem: PyPI
summary: Vulnerability in django
details: An issue was discovered in Django 2.2 before 2.2.13 and 3.0 before 3.0.7.
  Query parameters generated by the Django admin ForeignKeyRawIdWidget were not properly
  URL encoded, leading to a possibility of an XSS attack.
affects:
  ranges:
  - type: ECOSYSTEM
    introduced: "2.2"
    fixed: 2.2.13
  - type: ECOSYSTEM
    introduced: "3.0"
    fixed: 3.0.7
references:
- type: WEB
  url: https://www.djangoproject.com/weblog/2020/jun/03/security-releases/
- type: WEB
  url: https://groups.google.com/forum/#!msg/django-announce/pPEmb2ot4Fo/X-SMalYSBAAJ
- type: WEB
  url: https://docs.djangoproject.com/en/3.0/releases/security/
- type: WEB
  url: https://usn.ubuntu.com/4381-1/
- type: WEB
  url: https://usn.ubuntu.com/4381-2/
- type: WEB
  url: https://security.netapp.com/advisory/ntap-20200611-0002/
- type: WEB
  url: https://www.debian.org/security/2020/dsa-4705
- type: WEB
  url: https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/4A2AP4T7RKPBCLTI2NNQG3T6MINDUUMZ/
- type: WEB
  url: https://www.oracle.com/security-alerts/cpujan2021.html
aliases:
- CVE-2020-13596
modified: "2021-01-20T15:15:00Z"
published: "2020-06-03T14:15:00Z"
